Library/Homebrew/formulary.rb

```diff
@@ -57,6 +57,7 @@ def self.class_s(name)
     class_name = name.capitalize
     class_name.gsub!(/[-_.\s]([a-zA-Z0-9])/) { $1.upcase }
     class_name.tr!("+", "x")
+    class_name.gsub!(/\b[@]\b/, "AT")
```

Comment:
Capture parts of the matched text and use them in the replacement:

```ruby
"openssl@1.1".gsub!(/(.)@(\d)/, "\\1AT\\2")
```

(And a `sub!` should be sufficient since we don't expect to have multiple `@` in a formula name for now.)